Flaky test case: test_e2e_23_flow_manager.py::TestE2EFlowManager::test_065_install_flow

This test `tests/test_e2e_23_flow_manager.py::TestE2EFlowManager::test_065_install_flow` has been flaky lately, it has failed more than 3 times over the past days.

Looks like the original intention was to install flows in parallel, and there's a time sleep, maybe we can try to make this more stable by waiting for the result of the request, maybe the process is taking longer to start considering the sleep for 10s ( > 3x `STATS_INTERVAL`) 

```
=================================== FAILURES ===================================
___________________ TestE2EFlowManager.test_065_install_flow ___________________
self = <tests.test_e2e_23_flow_manager.TestE2EFlowManager object at 0x7f71d054b880>
    def test_065_install_flow(self):
        """
        Tests the performance and race condition
        with the creation of multiple flows
        """
        import subprocess
    
        api_url = KYTOS_API + '/flow_manager/v2/flows/00:00:00:00:00:00:00:01'
    
        import shlex
        for i in range(100, 200):
            cmd = '''curl -X POST -H 'Content-type: application/json' ''' + api_url + '''
                    -d "{\\"flows\\": [{\\"priority\\": 100, \\"cookie\\": 84114904,
                    \\"match\\": {\\"in_port\\": 1, \\"dl_vlan\\":''' + str(i) + '''},
                    \\"actions\\": [{\\"action_type\\": \\"output\\", \\"port\\": 2}]}]}"'''
    
            args = shlex.split(cmd)
            process = subprocess.Popen(args, shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
    
        # wait for the flow to be installed
        time.sleep(10)
    
        s1 = self.net.net.get('s1')
        flows_s1 = s1.dpctl('dump-flows')
    
>       assert len(flows_s1.split('\r\n ')) == 101
E       assert 78 == 101
E        +  where 78 = len([' cookie=0xab00000000000001, duration=25.370s, table=0, n_packets=16, n_bytes=672, priority=1000,dl_vlan=3799,dl_type...on=11.[96](https://gitlab.ampath.net/kytos/kytos-end-to-end-tester/-/jobs/22692#L96)1s, table=0, n_packets=0, n_bytes=0, priority=100,in_port="s1-eth1",dl_vlan=104 actions=output:"s1-eth2"', ...])
E        +    where [' cookie=0xab00000000000001, duration=25.370s, table=0, n_packets=16, n_bytes=672, priority=1000,dl_vlan=37[99](https://gitlab.ampath.net/kytos/kytos-end-to-end-tester/-/jobs/22692#L99),dl_type...on=11.961s, table=0, n_packets=0, n_bytes=0, priority=[100](https://gitlab.ampath.net/kytos/kytos-end-to-end-tester/-/jobs/22692#L100),in_port="s1-eth1",dl_vlan=[104](https://gitlab.ampath.net/kytos/kytos-end-to-end-tester/-/jobs/22692#L104) actions=output:"s1-eth2"', ...] = <built-in method split of str object at 0x1a1ed40>('\r\n ')
E        +      where <built-in method split of str object at 0x1a1ed40> = ' cookie=0xab00000000000001, duration=25.370s, table=0, n_packets=16, n_bytes=672, priority=1000,dl_vlan=3799,dl_type=...ation=0.005s, table=0, n_packets=0, n_bytes=0, priority=100,in_port="s1-eth1",dl_vlan=176 actions=output:"s1-eth2"\r\n'.split
tests/test_e2e_23_flow_manager.py:1047: AssertionError
```

--- a/tests/test_e2e_23_flow_manager.py
+++ b/tests/test_e2e_23_flow_manager.py
@@ -4,6 +4,7 @@ import time
 import pytest
 import requests
 
+from concurrent.futures import ThreadPoolExecutor, as_completed
 from tests.helpers import NetworkTest
 
 CONTROLLER = '127.0.0.1'
@@ -1024,19 +1025,33 @@ class TestE2EFlowManager:
         Tests the performance and race condition
         with the creation of multiple flows
         """
-        import subprocess
-
         api_url = KYTOS_API + '/flow_manager/v2/flows/00:00:00:00:00:00:00:01'
 
-        import shlex
-        for i in range(100, 200):
-            cmd = '''curl -X POST -H 'Content-type: application/json' ''' + api_url + '''
-                    -d "{\\"flows\\": [{\\"priority\\": 100, \\"cookie\\": 84114904,
-                    \\"match\\": {\\"in_port\\": 1, \\"dl_vlan\\":''' + str(i) + '''},
-                    \\"actions\\": [{\\"action_type\\": \\"output\\", \\"port\\": 2}]}]}"'''
+        def flow_request(dl_vlan):
+            payload = {
+              "flows": [{
+                "priority": 100,
+                "cookie": 84114904,
+                "match": {
+                  "in_port": 1,
+                  "dl_vlan": int(dl_vlan),
+                },
+                "actions": [{
+                  "action_type": "output",
+                  "port": 2
+                }]
+              }]
+            }
+            return requests.post(api_url, data=json.dumps(payload),
+                                 headers={'Content-type': 'application/json'})
 
-            args = shlex.split(cmd)
-            process = subprocess.Popen(args, shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        with ThreadPoolExecutor(max_workers=100) as executor:
+            futures = [
+                executor.submit(flow_request, dl_vlan) for dl_vlan in range(100, 200)
+            ]
+            for future in as_completed(futures):
+                response = future.result()
+                assert response.status_code == 202, response.text
 
         # wait for the flow to be installed
         time.sleep(10)

